filters/tee/tee.go

```diff
@@ -0,0 +1,172 @@
+package tee
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
```

Comment:
in new code, we tend to use the "CodeReviewComments": https://github.com/golang/go/wiki/CodeReviewComments#imports
for imports. Here, the logrus and filters imports could be below the stdlib ones, separated by an empty line. This is, of course, not obvious when just generally looking at the whole code base.